pkg/landscaper/controllers/installations/controller.go

```diff
@@ -131,6 +131,15 @@ func (c *Controller) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	}
 
 	if lsv1alpha1helper.IsCompletedInstallationPhase(inst.Status.Phase) && inst.Status.ObservedGeneration == inst.Generation {
+		// check whether the current phase does still match the combined phase of subinstallations and executions
+		op, err := c.initPrerequisites(ctx, inst)
```

Comment:
why do we need to init all prerequisites?
This is a expensive function that resolves the component descriptor and blueprint.
I think we should rather only init them when we need to go to succeeded phase?

Reply:
I removed the requirement for the operation